src/components/savings/index.js

```diff
@@ -2,3 +2,4 @@ export { default as APYPill } from './APYPill';
 export { default as SavingsSheetEmptyState } from './SavingsSheetEmptyState';
 export { default as SavingsPredictionStepper } from './SavingsPredictionStepper';
 export { default as SavingsSheetHeader } from './SavingsSheetHeader';
+export { default as SavingsListHeader } from './SavingsListHeader';
```

Comment:
Can we tidy up all the places that currently import `SavingsListHeader`

Reply:
I added this line, so I've tidied this up 👍